firmware/components/micropython/component.mk

```diff
@@ -265,6 +265,11 @@ MP_EXTRA_INC += -I$(PROJECT_PATH)/components/libnmea/src/nmea
 MP_EXTRA_INC += -I$(PROJECT_PATH)/components/libnmea/src/parsers
 endif
 
+ifdef CONFIG_MICROPY_USE_AM2320
```

Comment:
Just use your drivers CONFIG_DRIVER_AM2320_ENABLE here, don't add extra complexity if it's not usefull

Reply:
Changed to use the driver enable